framework/src/main/java/org/checkerframework/framework/type/typeannotator/TypeAnnotator.java

```diff
@@ -1,12 +1,18 @@
 package org.checkerframework.framework.type.typeannotator;
 
+import javax.lang.model.element.Element;
 import org.checkerframework.framework.type.AnnotatedTypeFactory;
+import org.checkerframework.framework.type.AnnotatedTypeMirror;
+import org.checkerframework.framework.type.AnnotatedTypeMirror.AnnotatedExecutableType;
 import org.checkerframework.framework.type.visitor.AnnotatedTypeScanner;
 
 /**
  * {@link TypeAnnotator} is an abstract AnnotatedTypeScanner to be used with {@link
  * ListTypeAnnotator}.
  *
+ * <p>When implementing this class, each method should return {@code null} instead of calling super,
```

Comment:
New line 36 calls super, contradicting this advice. Is the advice or the implementation incorrect?